xrdsst/core/util.py

```diff
@@ -8,19 +8,23 @@
 
 
 def get_admin_credentials(security_server, config):
-    admin_credentials_env_variable = security_server["admin_credentials"] if security_server.get("admin_credentials", "") else config["admin_credentials"]
+    admin_credentials_env_variable = security_server["admin_credentials"] if security_server.get("admin_credentials",
```

Comment:
Why have you changed this file?

Reply:
i don't remember change this, probably it was without notice